### DISCUSSION TITLE
6D100C19-C902-3BCF-E9DC-81621177663D

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#192] [General]
# 6D100C19-C902-3BCF-E9DC-81621177663D

The James Webb Space Telescope (JWST) is a joint NASA/ESA/CSA observatory that is planned to succeed the Hubble Space Telescope as NASA's flagship astrophysics mission. After many years of delays, it finally lifted off the ground on 2021-Dec-25 atop an Ariane 5 (ECA) rocket from Kourou, French Guiana. It arrived on the Sun-Earth Lagrange point-2 on January 2022, and has been continuously observing the cosmos since June 2022

https://celestia.mobi/addon?item=6D100C19-C902-3BCF-E9DC-81621177663D

**Comment by Vendrel** [comment]
Thank you! :)
